fvm/evm/testutils/backend.go

```diff
@@ -60,7 +60,7 @@ func ConvertToCadence(data []byte) []cadence.Value {
 }
 
 func fullKey(owner, key []byte) string {
-	return string(owner) + "~" + string(key)
+	return fmt.Sprintf("%x~%s", owner, key)
```

Comment:
Both `owner` & `key` are of type `[]byte`, so why the `"%x~%s"` format directive? :thinking:

Reply:
Because when I'm comparing the state diff, `%x` could show the proper Account address.

Reply:
Ah, right, all good then :ok_hand: